### ISSUE TITLE
Better Dynamic Data Menu & ACF Organization Options 

### ISSUE THREAD

**Issue by Gibsonion** [soflyy/oxygen-bugs-and-features#1562]
The Dynamic Data menu would benefit from an update in several areas. First the ACF menu layout gets cluttered quickly beyond a handful of datapoints, making it difficult to find things, there needs to be a way to organize and/or search the available options.

ACF best naming practises suggests: 
> **Don’t Repeat Names in Your Repeater**
> Avoid repeating the parent field’s name within sub field’s. Doing so not only causes long-winded field names for you, but also causes ACF to generate even longer meta keys in your wp_postmeta table. This is because ACF joins together both the parent and child field name’s to create the meta key like so **$parent_$index_$child.**

https://www.advancedcustomfields.com/blog/best-practices-designing-custom-fields/

Oxygen's only shows the subfield name, so you can't tell subfields with the same name apart, this could be fixed with a hover overlay that shows fields parent relationships and other useful info. 

Second a dropdown menu of Oxygen's available custom PHP variables, or even a list of shortcodes and their useage in the user guide.

In general the dynamic data UI would really benefit from a visual overhaul.

**Use cases**
Easier to work with dynamic data in templates.

Thanks

**Comment by silvenna** [comment]
+1 This would make life so much easier!

**Comment by HmCody** [comment]
An additional point, the form for inserting Custom Field/Meta Options Options has a funky double scroll bar when you click the drop box for "Key." See the attached image. Could you just expand the space for the dropdown itself so that the second scroll bar doesn't appear? It would significantly improve the user friendliness of the dialog. In case it matters, I am using Chrome to view this.

![image](https://user-images.githubusercontent.com/33326939/131895353-808f035c-2f96-4cbd-8628-670edbde91f0.png)

**Comment by netdoctor** [comment]
Makes total sense. Oxygen please :)

**Comment by tilen-seliskar** [comment]
Yes please. And do include all custom data (not just ACF but Metabox  and others as well).
